packages/manager/modules/vps/src/dashboard/cancel-resiliation/vps-cancel-resiliation.routing.js

```diff
@@ -0,0 +1,83 @@
+import snakeCase from 'lodash/snakeCase';
+import { BillingService } from '@ovh-ux/manager-models';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('vps.detail.dashboard.cancel-resiliation', {
+    url: '/cancel-resiliation',
+    views: {
+      modal: {
+        component: 'billingCancelResiliation',
+      },
+    },
+    layout: 'modal',
+    translations: { value: ['.'], format: 'json' },
+    resolve: {
+      cancelResiliation: /* @ngInject */ (
+        $http,
+        $q,
+        engagement,
+        setReactivateEngagementStrategy,
+      ) => (service) => {
+        if (engagement) {
+          return setReactivateEngagementStrategy();
+        }
+        service.cancelResiliation();
+        return $http
+          .put(
+            '/sws/billing/autorenew/services/update',
```

Comment:
can't resiliation be done through service direct PUT /serviceInfos ?

Reply:
Handled using the apiv6 `PUT /serviceInfos` call. Thanks!